QuasarHelios2015/src/org/team1540/quasarhelios/ControlInterface.java

```diff
@@ -62,7 +62,7 @@ private static void setupRollers() {
             Rollers.direction.set(Rollers.OUTPUT);
         });
 
-        Rollers.closed.toggleWhen(EventMixing.combine(povLeft, povRight));
+        Rollers.closed.toggleWhen(EventMixing.combine(povLeft, povRight, Igneous.joystick1.getButtonSource(5)));
```

Comment:
I think Rushdi wants a button to open/close the rollers, since that has to be coordinated with driving